ui/app/adapters/secret.js

```diff
@@ -34,8 +25,8 @@ export default ApplicationAdapter.extend({
     return this.ajax(this.urlForSecret(snapshot.attr('backend'), id), 'DELETE');
   },
 
-  urlForSecret(backend, id, infix = 'data') {
-    let url = `${this.buildURL()}/${backend}/${infix}/`;
+  urlForSecret(backend, id) {
+    let url = this.buildURL() + '/' + backend + '/';
```

Comment:
curious why the template string didn't work here.

Reply:
It did! There's some weird git shuffling and adding then moving back of files, so the concat is just the old way we did it and the template is now in a different file.